plugin/ochttp/span_annotating_trace.go

```diff
@@ -27,9 +27,16 @@ type spanAnnotator struct {
 	sp *trace.Span
 }
 
-// NewSpanAnnotator returns a httptrace.ClientTrace which annotates all emitted
-// httptrace events on the provided Span.
-func NewSpanAnnotator(_ *http.Request, s *trace.Span) *httptrace.ClientTrace {
+// TODO: Remove NewSpanAnnotator at the next release.
+
+// Deprecated: Use NewSpanAnnotatingTrace instead
+func NewSpanAnnotator(r *http.Request, s *trace.Span) *httptrace.ClientTrace {
+	return NewSpanAnnotatingTrace(r, s)
+}
+
+// NewSpanAnnotatingTrace returns a httptrace.ClientTrace which annotates all
+// emitted httptrace events on the provided Span.
+func NewSpanAnnotatingTrace(_ *http.Request, s *trace.Span) *httptrace.ClientTrace {
```

Comment:
NewSpanAnnotatingClientTrace

What is returned is not a trace, a client trace.